### PR TITLE
Provider can set selectable school property

### PR DESCRIPTION
**Pull request by inulty-dfe**
## Context

Providers have the ability to change the setting which shows/hides the School Placements page to candidates when they are applying of any of their courses.

This only affects the courses for which the provider is the training provider.

## Changes proposed in this pull request

Add provider controls for turning on and off the school placement selection in apply and also showing / hiding the link on Find courses that shows the list of potential placement schools available.

## Guidance to review

[Screencast from 25-09-24 09:01:00.webm](https://github.com/user-attachments/assets/4d235a67-1620-44fe-a797-958aa0b54d3f)

## Things to check

- [ ] This code does not rely on migrations in the same Pull Request
- [ ] If this code includes a migration adding or changing columns, it also backfills existing records for consistency
- [ ] If this code adds a column to the DB, decide whether it needs to be in analytics yml file or analytics blocklist
- [ ] API release notes have been updated if necessary
- [ ] Required environment variables have been updated added to the Azure KeyVault
- [ ] Inform data insights team due to database changes
- [ ] Make sure all information from the Trello card is in here
- [ ] Rebased main
- [ ] Cleaned commit history
- [ ] Tested by running locally
- [ ] Attach PR to Trello card

### CODE REVIEW

**Comment by inulty-dfe** [comment]
[Add UI in Publish to allow Providers to update selectable schools](https://trello.com/c/2gwfWa1Z/2368-add-ui-in-publish-to-allow-providers-to-update-selectable-schools)

File: app/views/publish/providers/school_placements/edit.html.erb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+      <%= f.govuk_error_summary %>
+
+      <h1 class="govuk-heading-l">
+        <span class="govuk-caption-l"><%= @provider.provider_name %></span>
+        <%= page_title %>
+      </h1>
+
+      <%= f.govuk_radio_buttons_fieldset(:selectable_school,
+                                         legend: { text: "Let candidates select their preferred schools when applying" }) do %>
```
Comment (CatalinVoineag) [inline app/views/publish/providers/school_placements/edit.html.erb:25]: Is it worth adding the text in this view in translations?

File: app/views/publish/providers/details.html.erb [hunk 2]
```diff
@@ -27,6 +27,9 @@
   Organisation details
 </h1>
 
+<h2 class="govuk-heading-m">
+  About your organisation
```
Comment (CatalinVoineag) [inline app/views/publish/providers/details.html.erb:31]: Is it worth adding the text in this view in translations?

**Review by CatalinVoineag — APPROVED** [review]
Just some translation questions. I don't really know what's the practice here on using translations. Not a blocker though. Looks Good

**Pull request merged**